load-balancer/response-time/src/main/java/io/smallrye/stork/loadbalancer/leastresponsetime/LeastResponseTimeLoadBalancerProvider.java

```diff
@@ -1,19 +1,20 @@
 package io.smallrye.stork.loadbalancer.leastresponsetime;
 
-import io.smallrye.stork.LoadBalancer;
-import io.smallrye.stork.ServiceDiscovery;
-import io.smallrye.stork.config.LoadBalancerConfig;
+import io.smallrye.stork.api.LoadBalancer;
+import io.smallrye.stork.api.ServiceDiscovery;
+import io.smallrye.stork.api.config.LoadBalancerAttribute;
+import io.smallrye.stork.api.config.LoadBalancerName;
 import io.smallrye.stork.spi.LoadBalancerProvider;
 
-public class LeastResponseTimeLoadBalancerProvider implements LoadBalancerProvider {
+@LoadBalancerName("least-response-time")
+@LoadBalancerAttribute(name = "force-retry-threshold", defaultValue = "1000", description = "after how many calls should a service instance reuse be forced on no failures")
```

Comment:
The format looks great - pretty close to the Reactive Messaging one. 

In RM, the description can use the Markdown syntax. It complexifies a bit the work to do in Quarkus (because it expects HTML), but works well once you transform the text to HTML.

Don't make the same mistake I did - always indicate the time unit in the description. I really would like to be able to reuse the `Duration` support.

Reply:
we can parse duration, we have means for it, but all the config options are currently Strings. So ServiceDiscoveryProvider, etc has to parse the string to duration itself.

Reply:
Also, i copy the description directly to a markdown table so in some way markdown is supported here. But it is not translated to HTML/javadoc for javadocs

Reply:
On the quarkus side, we would need to register config attributes (for tooling and dev ui). These requires HTML description. But no worries, we have done that work for reactive messaging.